CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.6.0] - 2020-10-29
+- added async submit for `TextConcatType` to update list while typing in field
```

Comment:
TextType fehlt noch ;)
Grundsätzlich gehören Changelogs allerdings nicht in Pull-Request, damit es keine merge-konflikte gibt